fms/training/plugins.py

```diff
@@ -206,26 +211,34 @@ class Checkpointer(TrainerPlugin):
     """
     A training plugin to write checkpoints.
     TODO: This will require changes to handle distributed checkpoints.
+
+    Args:
```

Comment:
This docstring is missing a dataset param. Also, not a big deal, but maybe should be ordered in same order as params of __init__.